src/steps/doBackportVersions.ts

```diff
@@ -171,19 +175,26 @@ async function resolveConflictsOrAbort(owner: string, repoName: string) {
   }
 }
 
-function getPullRequestTitle(baseBranch: string, commits: Commit[]) {
-  const commitMessages = commits
+function getPullRequestTitle(
+  baseBranch: string,
+  commits: Commit[],
+  prTitle: string
+) {
+  // @ts-ignore
```

Comment:
Had to disable ts and lint for this line. It will be used in the `eval` call if the consumer defines it in their config